apis/traffic-access/v1alpha4/traffic-access.md

```diff
@@ -0,0 +1,280 @@
+# Traffic Access Control
```

Comment:
This file should be moved to /apis/traffic-access/traffic-access.md

Reply:
git moved the file as instructed.